charts/camunda-platform/Chart.yaml

```diff
@@ -23,12 +23,12 @@ dependencies:
     alias: identityKeycloak
     repository: https://charts.bitnami.com/bitnami
     version: 19.4.1
-    condition: "identityKeycloak.enabled"
+    condition: "identity.keycloak.enabled,identityKeycloak.enabled"
```

Comment:
@jessesimpson36 According to Helm docs, this will not work.

https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies

> Only the first valid path found in the list is evaluated and if no paths exist then the condition has no effect.

Reply:
This is intentional. `identity.keycloak.enabled` will be `nil` if you're using the `identityKeycloak` values.  and when it evaluates to `nil`,  `identityKeycloak.enabled` will be used.

the compatibility layer is not rendered at the time that `Chart.yaml` conditions are evaluated.

There was a previous PR where I messed up with this usage of the comma operator, but I feel more confident about this one.

Reply:
As I remember there was a discussion in the Helm upstream repo if the key is not found doesn't mean it will be `nil`.
But if you are confident about it no need to dig more.

Reply:
Yesterday I wrote this automated test 
https://github.com/camunda/camunda-platform-helm/blob/969dc01d208620b0412f260a7d41d858e1f4038b/charts/camunda-platform/test/unit/identity/secret_test.go#L80-L135

And I tested a couple different situations by tweaking this test, such as without my Chart.yaml fix, or without this `identity.postgresql.enabled` condition.  I feel more confident now that this works, but if I'm wrong, I think with that unit test, it'd be easier for you create a scenario where this condition could be faulty... If you wanted to look further into it.